WebhookTemplate.AzureFunction/Functions/Subscriber.cs

```diff
@@ -0,0 +1,305 @@
+using IdentityModel.Client;
+using IdentityModel;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.IdentityModel.Tokens;
+using Newtonsoft.Json;
+using Profisee.WebhookTemplate.Common.Dtos;
+using System.Collections.Generic;
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.IO;
+using System.Net.Http;
+using System.Security.Cryptography;
+using System.Text.Json;
+using System.Threading.Tasks;
+using System.Net.Http.Headers;
+using Profisee.WebhookTemplate.Common.Configuration;
+using Profisee.WebhookTemplate.Common.Clients.Dtos;
+using Profisee.WebhookTemplate.Common.Clients.Entities.Responses;
+using Profisee.WebhookTemplate.Common.Clients;
+using System.Net.Mime;
+using System.Text;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+
+namespace WebhookTemplate.AzureFunction.Functions;
+
+public class Subscriber
+{
+    [FunctionName("Subscriber")]
+    public async Task<IActionResult> Run([HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = null)] HttpRequest req,
+         ILogger log)
+    {
+        var config = new ConfigurationBuilder()
+                 .AddJsonFile("local.settings.json", optional: true, reloadOnChange: true)
+                 .AddEnvironmentVariables()
+                 .Build();
+        var client = new HttpClient(this.getHttpHandler());
+        var url = config.GetValue<string>("ServiceUrl") + "rest/";
+        client.BaseAddress = new Uri(url);
+        string msg;
+        try
+        {
+             msg = $"ActivityGeneric HTTP trigger function processed a request.";
```

Comment:
```suggestion
            msg = $"ActivityGeneric HTTP trigger function processed a request.";
```
I **think** there's an extra space here.

Reply:
Good catch, I removed that extra space